influxdb3_write/src/lib.rs

```diff
@@ -216,8 +216,8 @@ pub struct PersistedCatalog {
 /// The collection of Parquet files that were persisted in a snapshot
 #[derive(Debug, Serialize, Deserialize, Eq, PartialEq, Clone)]
 pub struct PersistedSnapshot {
-    /// The last file id used to store `ParquetFile`s with
-    pub last_file_id: u64,
+    /// The next file id to be used with `ParquetFile`s when the snapshot is loaded
+    pub next_file_id: u64,
```

Comment:
Any reason to not have this type be `ParquetFileId`?

Reply:
Oversight, I've updated it to use the correct type.